shap/plots/_scatter.py

```diff
@@ -214,31 +233,8 @@ def scatter(
     ind = convert_name(ind, shap_values_arr, feature_names)
```

Comment:
`ind` is set to 0 above 

![image](https://github.com/user-attachments/assets/80183ff2-f86d-434b-9881-87ca1c548f51)

so this call to `convert_name` is pretty much just dead code. (`convert_name` only does something if the input is a string)

Reply:
Good catch. I hadn't looked at the use of `ind` too closely, so I'll take another look and try to simplify that. It looks like this is just a hangover from code in `scatter_legacy` that was copied over, so it should be safe to remove now.